packages/langchain/src/orchestration/client.test.ts

```diff
@@ -129,4 +158,93 @@ describe('orchestration service client', () => {
       'Request failed with status code 400'
     );
   }, 1000);
+
+  it('supports streaming responses', async () => {
+    mockStreamInferenceWithResilience();
```

Comment:
[req] Why are we mocking responses with resilience? Why is resilience needed in this test? Can we not just call `mockInference()`?

Reply:
Done.